src/ResourceManager/Automation/Commands.Automation.Test/Commands.Automation.Test.csproj

```diff
@@ -233,8 +249,14 @@
     </ProjectReference>
   </ItemGroup>
   <ItemGroup>
+    <None Include="app.config" />
```

Comment:
Why do you need app.config?  If you take the latest changes, you should not

Reply:
Removed app.config